www/.eleventy.js

```diff
@@ -47,6 +47,10 @@ module.exports = function (eleventyConfig) {
     return JSON.stringify(obj);
   });
 
+  eleventyConfig.addFilter('toDocPath', function (str) {
+    return str.replace(/^\.\/\_template/, `${process.env.VERCEL_GIT_COMMIT_REF || 'main'}/docs`)
```

Comment:
@FredKSchott added this filter to transform the Eleventy input path back to the repo path! Also pulls in the correct branch name from Vercel

Reply:
niiiiiice 👍 👍 👍 👍